src/net.h

```diff
@@ -1176,12 +1176,17 @@ class CNode
         m_addr_known->insert(_addr.GetKey());
     }
 
+    bool IsAddrCompatible(const CAddress& addr) const
+    {
+        return m_wants_addrv2 || addr.IsAddrV1Compatible();
+    }
+
     void PushAddress(const CAddress& _addr, FastRandomContext &insecure_rand)
     {
         // Whether the peer supports the address in `_addr`. For example,
         // nodes that do not implement BIP155 cannot receive Tor v3 addresses
         // because they require ADDRv2 (BIP155) encoding.
-        const bool addr_format_supported = m_wants_addrv2 || _addr.IsAddrV1Compatible();
+        const bool addr_format_supported = IsAddrCompatible(_addr);
```

Comment:
perhaps hoist the documentation too
```diff
+    /**
+     * Whether the peer supports the address. For example, a peer that does not
+     * implement BIP155 cannot receive Tor v3 addresses because it requires
+     * ADDRv2 (BIP155) encoding.
+     */
     bool IsAddrCompatible(const CAddress& addr) const
     {
         return m_wants_addrv2 || addr.IsAddrV1Compatible();
@@ -1183,9 +1188,6 @@ public:
 
     void PushAddress(const CAddress& _addr, FastRandomContext &insecure_rand)
     {
-        // Whether the peer supports the address in `_addr`. For example,
-        // nodes that do not implement BIP155 cannot receive Tor v3 addresses
-        // because they require ADDRv2 (BIP155) encoding.
         const bool addr_format_supported = IsAddrCompatible(_addr);
```

Reply:
Good idea, done.